openedx/features/edly/utils.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.sites.shortcuts import get_current_site
+from django.db.models import Q
+
+from openedx.features.edly.models import EdlySubOrganization
+
+
+def user_has_edly_organization_access(request):
+    """
+    Returns True if the edly-sub-organization of the requested URL belongs to the user's linked edly-sub-organizations.
+
+    Arguments:
+        request: HTTP request object
+
+    Returns:
+        bool: Returns True if User has Edly Organization Access Otherwise False.
+    """
+    if not request.user.edly_profile:
+        return False
+
+    current_site = get_current_site(request)
+    edly_sub_org = EdlySubOrganization.objects.filter(Q(lms_site=current_site) | Q(studio_site=current_site)).first()
```

Comment:
use single `count` query using `request.site` and if the user is linked to that edly sub organization.

Reply:
`request.site ` is note available in cases for example in case of `Request Factory`. `get_current_site` is a Django provided function available for getting site from request.

Reply:
`'WSGIRequest' object has no attribute 'site'` that's why using `Django` provided helper method  `get_current_site `

Reply:
Feedback Implemented